ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/table-viz.view.js

```diff
@@ -55,14 +55,26 @@ function getFilenameFromContentDisposition(header) {
     return parts[1];
 }
 
+const adaptFetchData = async (response) => ({
+    data: await response.text(),
+    status: response.status,
+    getResponseHeader: (requestedHeader) => {
+        for(const header of response.headers.entries()) {
+            if (!Array.isArray(header)) {
+              continue
+            }
+            if ((header[0] || '').toLowerCase() === (requestedHeader || '').toLowerCase())
+                return header[1];
```

Comment:
❓ If headers can be multivalued I'd suggest refactoring this into `getResponseHeaders()` and `getFirstResponseHeader()` to avoid hiding this detail.